### ISSUE TITLE
Issue with reading files created on other operating systems

### ISSUE THREAD

**Issue by DarthJDG** [booxood/readlines#18]
The library should properly detect the newlines used in the file, as it might come from a different OS. It's used for reading files, not creating them. In my opinion it's bad design to assume the current OS's newline format.

**Comment by booxood** [comment]
You are right. It is bad design that default use current OS's newline separator.

Are you idea about how determine newline separator?
